recipe/meta.yaml

```diff
@@ -1,29 +1,31 @@
 {% set name = "dash-mantine-components" %}
-{% set version = "0.15.0" %}
+{% set version = "0.15.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dash_mantine_components-{{ version }}.tar.gz
-  sha256: 9f7277c0eae63f046a0a06706ba9a8ce77af48b1fdf351c489eaee75655b9aa3
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/dash_mantine_components-{{ version }}.tar.gz
+  sha256: bb24153af09ccc42972af397101ee62718ebe2bbf29fa7ed33c02997059352bd
 
 build:
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
 
 requirements:
   host:
-    - python >=3.7
+    - python {{ python_min }}
     - pip
     - setuptools
   run:
-    - python >=3.7
+    - python >={{ python_min }}
     - dash >=2.5.1
 
 test:
+  requires:
+    - python {{ python_min }}
```

Comment:
Is this required, isn't python a run_export anyway?

Reply:
Hi @0xbe7a 
I believe this fine - its from CFEP-25
You can take a look at - https://github.com/conda-forge/dash-mantine-components-feedstock/pull/20
Details - This PR updates the recipe to use the noarch: python syntax as described in [CFEP-25](https://github.com/conda-forge/cfep/blob/main/cfep-25.md). Please see our [documentation](https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python) for more details.
Thank you.